src/modules/users/models/order.interace.ts

```diff
@@ -0,0 +1,19 @@
+import { ObjectId } from "mongodb";
+
+export interface OrderItem {
+    medicine: ObjectId;
+    quantity: number;
+    price: number;
+    total: number;
+}
+
+export interface OrderModel {
+    user: ObjectId; //Might need to be string instead
```

Comment:
if we're using the userId from the mongo record, we can have it as objectId, if we switch to email, then it should be a string yeah. but should be okay to be an object id because that's the type that we're getting from mongo when we fetching the user by email.

Reply:
oh i see, yeah let's leave it like this for now and we'll look into how to do it from the front end